packages/quantic/force-app/main/default/lwc/quanticSummary/e2e/quanticSummary.e2e.ts

```diff
@@ -0,0 +1,75 @@
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+import {testSearch, testInsight, expect} from './fixture';
+
+const fixtures = {
+  search: testSearch as typeof testSearch,
+  insight: testInsight as typeof testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+  test.describe(`quantic-summary ${useCase.label}`, () => {
+    test.describe('when selecting result per page', () => {
+      test('should display the correct per-page number and log analytics', async ({
```

Comment:
That's not really a function of the quanticSummary.

The whole search/analytics part should already be tested in the `quanticResultsPerPage` component.

And what gets displayed in the quanticSummary is really more of a unit test than an e2e test.

Reply:
ahh ok, j'avoue que le summary lui meme ne log rien.. bon je vais deleter ca lol